pkgs/development/tools/fdroidserver/default.nix

```diff
@@ -0,0 +1,59 @@
+{ pythonPackages
```

Comment:
```suggestion
{
```

Reply:
I don't get it. Where does pythonPackages come from then?

Reply:
You can use `python3.pkgs`.